### ISSUE TITLE
check whether we wanna allow assigment on task (measure, etc) properties

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr3#1149]
at least in the "weights PR" properties became an AB. where assignment was possible. 
I don't like that. props should be read-only? as static as possible?

**Comment by mb706** [comment]
I agree that allowing assignment to properties may be a bad idea here; it is not entirely clear what 'removing' a property should do. E.g. for weights: remove the associated col roles? Or just signal that the relevant col role should be ignored? For missings: remove features with missing values?

**Comment by be-marc** [comment]
We agree with Martin.
